diff --git a/node_modules/dmg-builder/vendor/mac_alias/alias.py b/node_modules/dmg-builder/vendor/mac_alias/alias.py
index fc017af..cfa8919 100644
--- a/node_modules/dmg-builder/vendor/mac_alias/alias.py
+++ b/node_modules/dmg-builder/vendor/mac_alias/alias.py
@@ -10,7 +10,6 @@ import os
 import os.path
 import stat
 import sys
-from unicodedata import normalize
 
 if sys.platform == 'darwin':
     from . import osx
@@ -390,10 +389,6 @@ class Alias (object):
         st = osx.statfs(path)
         vol_path = st.f_mntonname
 
-        # File and folder names in HFS+ are normalized to a form similar to NFD.
-        # Must be normalized (NFD->NFC) before use to avoid unicode string comparison issues.
-        vol_path = normalize("NFC", vol_path.decode('utf-8')).encode('utf-8')
-
         # Grab its attributes
         attrs = [osx.ATTR_CMN_CRTIME,
                  osx.ATTR_VOL_NAME,
